NEWS

```diff
@@ -1,18 +1,158 @@
 #
-## Copyright (C) Mellanox Technologies Ltd. 2001-2020.  ALL RIGHTS RESERVED.
+## Copyright (C) Mellanox Technologies Ltd. 2001-2021.  ALL RIGHTS RESERVED.
 ## Copyright (C) UT-Battelle, LLC. 2014-2019. ALL RIGHTS RESERVED.
-## Copyright (C) ARM Ltd. 2017-2020.  ALL RIGHTS RESERVED.
+## Copyright (C) ARM Ltd. 2017-2021.  ALL RIGHTS RESERVED.
 ##
 ## See file LICENSE for terms.
 ##
 #
 
-## Current
-### Features: TBD
-#### UCX Core
-- Added ucp_tag_msg_recv_nbx routine.
-#### UCX Java (API Preview) TBD
-### Bugfixes: TBD
+## 1.9.0 (February 1, 2021)
+### Features:
+#### Core
+* Added support for Nvidia HPC SDK
+* Added support for latest PGI and Clang
+* Added support for ROCM-3.7+ (warning generated if older version detected)
+#### Architecture
+* Added Arm SVE memcpy()
+* Redesigned Arm WFE support
+* Improved clear_cache performance for Arm
+* Added architecture detection for Zhaoxin CPU
+#### CI
+* Added release builds on CUDA 11
+* Enabled performance validation in gtest
+#### UCP
+* Added put/offload/short and put/offload/zcopy protocols
+* Added receive message nbx routine
+* Reworked AM implementation and API, which adds support for RNDV semantics
+* Added support for multi-lane connection manager over TCP
+* Added support for printing AM tls with info log level
+* Implement flush and destroy for UCT EPs on UCP worker
+* Reduced UCP request size
+* Added support for keepalive protocol
+* Added support for multi-fragment
```

Comment:
seems we have to be more specific in commit messages to use this new format.
For example this line is not clear (there are others too)

Reply:
More informative and accurate commit message.titles will help. These are very much manually filtered and refined version of the messages. If you have specific suggestion, please go ahead and post it.